styles.css

```diff
@@ -5,13 +5,12 @@
 .shoe4 { height: 100px; width: 180px; }
 .shoppingcart { width: 15px; }
 h1 { color: red; font-family: 'PT Sans Narrow', sans-serif;  font-size: 10px;  }
-ul { display: inline-block; }
-li { display: inline-block; color: red;  margin: 10px; }
```

Comment:
You had two sets of rules for `ul` and `li` in here, which you didn't need. I removed these two extra ones.